.github/workflows/build-and-test.yml

```diff
@@ -58,6 +61,10 @@ jobs:
             sdk: 6.0.x
             runtime: -x64
             codecov: false
+        exclude:
+          - isARM: false
```

Comment:
@JimBobSquarePants: I am not sure that I understand the intention of the `exclude` here, does that mean only the arm runners should be executed and all others should not run?

Reply:
The opposite. Exclude the ARM runners unless the label exists. 

I thought best to run the other runners at all times which helps us avoid any errors we potentially introduce with conditionals.